adafruit_platformdetect/board.py

```diff
@@ -629,6 +640,10 @@ def any_asus_tinker_board(self) -> bool:
     def any_stm32mp1(self) -> bool:
         """Check whether the current board is any stm32mp1 board."""
         return self.id in boards._STM32MP1_IDS
+    @property
+    def any_stm32mp157(self) -> bool:
```

Comment:
This is a duplicate of `any_stm32mp1()` which should just be able to be used.

Reply:
removed it